docs/basic_operation.rst

```diff
@@ -16,31 +18,31 @@ GUI-based Operation
 
 #. Pypot main window should display at this point.
 #. Click button for find potentiostat.
-#. In the drop down box the communication port that it is connected should be displayed after the program finds the correct port.
+#. In the drop down box the communication port that is connected should be displayed after the program finds the correct port.
 #. Now an experiment can be created by clicking on the add button
 #. Once clicked, a window should pop up to select the type of experiment that you want to run. Click on the appropriate experiment type.
 #. The experiment choice window should close and a new window should appear that allows you to edit the experiment parameters.
-#. Once experiment parameters input, click generate preview to show the time vs. voltage profile to verify that your experimental procedure appears correct.
+#. Once experiment parameters are input, click generate preview to show the time vs. voltage profile to verify that your experimental procedure appears correct.
 #. Once satisfied, click save experiment file and then save the file as a .yml file.
 #. Now you can begin the experiment by pressing the start button beside the start/stop experiment label.
-#. Once experiment is complete, click disconnect the disconnect potentiostat button.
+#. Once experiment is complete, click the disconnect potentiostat button.
 #. Now the potentiostat is safe to be removed and the program closed.
 
 Command-Line Operation
----------------------------
+----------------------
 
 .. rubric:: Experimental Setup
 
 #. Plug the USB cable into the computer and the potentiostat.
-#. Attach electrodes from potentiostat as appropriate to the experimental setup.
+#. Attach electrodes from potentiostat as appropriate to the experimental setup (if you are not familiar with how to set up an electrochemical experiment, see :ref:`setup`).
 
 .. rubric:: Program Setup
 
 #. Copy the example config file to an easy to navigate to directory.
 #. Edit this file to accomodate your experiment parameters.
```

Comment:
@aplymill7 Should we add saving instructions here? Do they just hit save, or should it be saved with a different name for future reference?

Reply:
It's saved based on the config file and a timestamp should be added as a unique identifier. So I don't think necessary to modify instructions here.